### PR TITLE
more speakers, and add eventbrite link

### PR DESCRIPTION
**Pull request by evykassirer**

### CODE REVIEW

File: _pages/speakers.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ permalink: /speakers/
 
 ### Julia Evans
 
+[so you want to be a wizard](/talks#so-you-want-to-be-a-wizard)
```
Comment (arshiamufti) [inline _pages/speakers.md]: title case this?

**Comment by arshiamufti** [comment]
overall lgtm (just a comment about julia's talk)

File: _pages/speakers.md (hunk 1 above)
Comment (arshiamufti) [reply _pages/speakers.md]: also is there no abstract for this yet?

File: _pages/talks.md [hunk 2]
```diff
@@ -8,7 +8,40 @@ permalink: /talks/
 
 # StarCon Talk Lineup
 
-The schedule will be announced soon.
+There will be 2 keynotes and 20 lightning talks at StarCon. The full lineup and order/schedule will be announced soon!
+
+## Keynotes
+
+### so you want to be a wizard
```
Comment (arshiamufti) [inline _pages/talks.md]: title case

Comment (evykassirer) [reply _pages/talks.md]: That's how she gave it to us so idk

**Pull request merged**